packages/cli/src/Config.ts

```diff
@@ -33,22 +31,6 @@ export class Plugin {
   }
 }
 
-function defaultIgnorePredicate(
-  path: string,
-  basename: string,
-  root: string,
-  type: EntryType
-): boolean {
-  return (
-    // ignore paths starting with a dot
-    basename.startsWith('.') ||
-    // ignore TypeScript declaration files
-    (basename.endsWith('.d.ts') && type === EntryType.File) ||
```

Comment:
I'm happy to add this hardcoding back in, but I actually don't know if this exclusion makes sense. Who says `.d.ts` files never need to be codemodded? :smile: